### PR TITLE
Styles patches for alert banner headings

### PR DESCRIPTION
**Pull request by ncksllvn**
Some extra style patches after the big style upgrades that came with the spacing/typography changes in department-of-veterans-affairs/vets-website/pull/6780.

Resolves department-of-veterans-affairs/vets-website/issues/6802

### CODE REVIEW

File: content/pages/health-care/health-conditions/mental-health.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ If you qualify for VA health care, you can get high-quality mental health servic
 
 <div class="usa-alert usa-alert-warning va-alert">
   <div class="usa-alert-body">
-	<h5>Are you in crisis? <a id="crisis-expander-link">You have options.</a></h5>
+	<h4 class="usa-alert-heading">Are you in crisis? <a id="crisis-expander-link">You have options.</a></h4>
```
Comment (phiden) [inline content/pages/health-care/health-conditions/mental-health.md]: Can we get a little more padding beneath, and/or shrink the icon a bit? 
screen shot 2017-11-17 at 10 58 42 am

File: content/pages/health-care/health-conditions/mental-health/depression.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Depression is a serious illness, but this common mental health problem is also h
 
 <div class="usa-alert usa-alert-warning va-alert">
   <div class="usa-alert-body">
-	<h5>Are you in crisis? <a id="crisis-expander-link">You have options.</a></h5>
+	<h4>Are you in crisis? <a id="crisis-expander-link">You have options.</a></h4>
```
Comment (phiden) [inline content/pages/health-care/health-conditions/mental-health/depression.md]: This one looks better though I couldn't tell you why!! 
screen shot 2017-11-17 at 10 59 59 am

File: content/pages/health-care/index.md [hunk 3]
```diff
@@ -51,8 +51,8 @@ With VA health care, you’re covered for regular checkups with your primary car
 
 <div class="va-alert usa-alert usa-alert-warning">
   <div class="usa-alert-body">
-  <h5 class="va-alert-title">Are you homeless or at risk of becoming homeless?<br><a id="crisis-expander-link">VA may be able to help</a>.
-  </h5>
+  <h4 class="va-alert-title">Are you homeless or at risk of becoming homeless?<br><a id="crisis-expander-link">VA may be able to help</a>.
```
Comment (phiden) [inline content/pages/health-care/index.md]: This one looks perfect from a padding perspective -- MST, substance abuse and PTSD could all also use more padding beneath the icon.

**Review by phiden — CHANGES_REQUESTED** [review]
The alert boxes that are a single line all need either: a little more padding beneath the icon OR a smaller icon, please

**Comment by ncksllvn** [comment]
@phiden PTAL

**Comment by phiden** [comment]
Before: 
screen shot 2017-11-17 at 12 50 44 pm

After:
screen shot 2017-11-17 at 12 50 31 pm

@melwoodard @ncksllvn if this looks ok to you then I think we're good to merge

**Comment by ncksllvn** [comment]
@phiden LGTM!

**Review by phiden — APPROVED** [review]

**Review by melwoodard — APPROVED** [review]
Looks good!

**Pull request merged**